### ISSUE TITLE
[Accessibility] [Supporting the platform -TSDocs]: TSDocs link is not visible properly in high-contrast dessert mode. [Sev3]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#41]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator

**Pre-requisites:**
Settings > Accessibility > Contrast themes > Select 'Dessert' mode.

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now, enable high-contrast dessert mode in settings and check whether the ‘TSDocs’ link is properly visible or not. 

**Actual:**
TSDocs link is not visible properly when high-contrast dessert mode is enabled.

**Expected:**
TSDocs link should be visible properly when high-contrast dessert mode is enabled.

**User Impact:**
When content or components are not visible properly in high contrast modes or they don’t appear as interactive elements, customers who rely on that feature are blocked from information and functionality of the full user experience within the site/app.

**Guideline Reference:**
EN 301 549 V3.2.1 - 11.6.2.0 No disruption of accessibility features

<img width="936" alt="MAS4 3 1_TSDocs link is not visible properly in high-contrast dessert mode" src="https://user-images.githubusercontent.com/88436727/151989327-28938bcf-ad62-41a4-b0b5-a3c50f2cb143.png">

**Comment by octogonz** [comment]
This issue has been closed because this Jekyll project has been retired. The documentation content has been migrated to Docusaurus. 

If the problem has not been addressed, please file a new issue under this GitHub project instead:  

https://github.com/microsoft/rushstack-websites
